### ISSUE TITLE
Incorrect Rendering of Tufte Style Output in R Markdown Document

### ISSUE THREAD

**Issue by Enrico68** [jalvesaq/Nvim-R#809]
Issue Body:
I am encountering an issue with rendering an R Markdown document using the Tufte style. Despite specifying the Tufte style in the YAML header, the output is not rendered as expected. Below are the steps to reproduce the issue, along with the YAML configuration and an extract of the initial lines of the file.

### Steps to Reproduce:
1. Create an R Markdown file with the following YAML header:

```yaml
---
title: "An Example Using the Tufte Style"
author: "John Smith"
output:
  tufte::tufte_html: default
```
- frist lines of `Rmd` file

```markdown
## Passo 1: Importazione delle librerie necessarie

In primo luogo, per eseguire il nostro processo di creazione di
strategie di trading, dobbiamo importare le librerie necessarie nel
nostro ambiente. In tutto questo processo, utilizzeremo alcune delle
librerie finanziarie più popolari in R, ovvero Quantmod, TTR e
Performance Analytics. Utilizzando la funzione library in R, possiamo
importare i nostri pacchetti richiesti.

```{r}
library(quantmod)
library(TTR)
library(PerformanceAnalytics)
```
---

3. Attempt to render the document using the specified Tufte style.

### Expected Behavior:
The document should be rendered with the Tufte style formatting, including side notes and other distinctive features of the Tufte style.

### Actual Behavior:
The document is rendered with default formatting, not reflecting the Tufte style specifications. I have tried using the commands `\kr` and `\ka`, but the output remains in the default style.

### Supporting Information:
- Screenshot of the incorrectly rendered output:

<img width="1042" alt="screenshot_04" src="https://github.com/jalvesaq/Nvim-R/assets/7678106/9503bca7-2821-4f4c-b820-da005ae4961a">

I would appreciate any guidance on how to resolve this issue and correctly render the document in the Tufte style. Thank you for your assistance.

**Comment by jalvesaq** [comment]
I can't replicate the issue. On the left, I used your YAML header, and on the right `output: html_document`:

![image](https://github.com/jalvesaq/Nvim-R/assets/891655/406e65af-2297-4e72-88db-37a26a7b5907)

I used `<LocalLeader>kr` in both cases to generate the output.

**Comment by Enrico68** [comment]
Thank you for your prompt response and for attempting to replicate the issue. I appreciate the comparison images you've provided.

I would like to provide some additional information that may be helpful. My operating system is macOS version 13.6.1, and I am using R version 4.2.3. Upon further testing, I have found that the issue seems to be isolated to the Tufte theme. Other themes and packages, such as `rmdformats`, `prettydoc`, `hrbrthemes`, and `tint`, are rendering as expected in my environment.

```R
library(rmdformats)
library(prettydoc)
library(hrbrthemes)
library(tint)
```

This leads me to believe that there might be a compatibility issue or a specific configuration requirement for the Tufte theme on my system. If there are any known issues with the Tufte theme on macOS 13.6.1 or any additional troubleshooting steps you could suggest, I would be grateful for your guidance.

Thank you for your assistance in resolving this matter.



Citations:
[1] https://pplx-res.cloudinary.com/image/upload/v1711872327/user_uploads/kgptjwmbx/immagine.jpg

**Comment by jalvesaq** [comment]
I'm on Linux, using R 4.3.3, and have no access to any Mac OS. I simply installed the tufte package and noted that the style was different from the default. You could try the same code in RStudio to see if you get the same result. If it works in RStudio, then, the bug is in Nvim-R; if not, the problem is something else in your configuration.

**Comment by Enrico68** [comment]
Upon further review of my R Markdown file, I have discovered an error in the original document that I had overlooked. It appears that my failure to thoroughly test the code snippets included in the file led to the issue I reported. The code, in fact, functions correctly when executed properly. I want to extend my sincerest apologies for any inconvenience this may have caused and express my gratitude for the time and effort you have invested in addressing my concern. Given this new information, I believe we can consider this matter resolved. Thank you once again for your support.
